frontend/src/components/MeetingRoom/ToolbarOverflowButton/ToolbarOverflowButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { ReactElement, useCallback, useRef, useState } from 'react';
+import { Tooltip } from '@mui/material';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import ToolbarButton from '../ToolbarButton';
+import ToolbarOverflowMenu from '../ToolbarOverflowMenu';
+
+/**
+ * ToolbarOverflowButton Component
+ *
+ * Displays a clickable button that opens a grid of hidden toolbar buttons for smaller viewport devices.
+ * @returns {ReactElement} - The ToolbarOverflowButton Component.
+ */
+const ToolbarOverflowButton = (): ReactElement => {
+  const anchorRef = useRef<HTMLButtonElement>(null);
+  const [open, setOpen] = useState<boolean>(false);
+  const [openEmojiGridMobile, setOpenEmojiGridMobile] = useState<boolean>(true);
+  const timer = useRef<number | undefined>(undefined);
+
+  const handleButtonToggle = () => {
+    setOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClickAway = useCallback(() => {
+    setOpen(false);
+    // We close the emoji grid so its animation is in-sync with the overflow menu animation.
+    setOpenEmojiGridMobile((prevValue) => {
+      if (timer.current === undefined) {
+        timer.current = window.setTimeout(() => {
+          setOpenEmojiGridMobile(prevValue);
+          timer.current = undefined;
+          // The default Transition timer is 250
+        }, 250);
```

Comment:
nit: is it worth putting the `250` as a constant in the utils file and importing it in here?

Reply:
If we change the animation timings for mui in the future, we can. Let's keep as-is for now since we only use it once.

Reply:
Oh, nice! Much simpler. Thanks Mike!